src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/MSBuildProcessManager.cs

```diff
@@ -26,6 +30,11 @@ internal static class MSBuildProcessManager
                 RedirectStandardOutput = true,
             };
 
+            if (localPackageCache != null)
+            {
+                processStartInfo.Environment.Add("NUGET_PACKAGES", localPackageCache);
+            }
```

Comment:
hmmm. what's going on here exactly?

Reply:
Makes it so that we don’t restore into the global cache because we have test packages.

Nate recommended we do it this way. It’s the pattern we follow on many similar “repos”

Reply:
Can we set `/p:NuGetPackageRoot` instead?

Reply:
I will give it a try, but I'm not sure it will work.

Reply:
Doesn't work. Reverting to old ways.